packages/atoms/src/Chip.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactNode } from 'react';
+import { Text } from './Text';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { TouchableOpacity } from 'react-native';
+import { Box } from './Box';
+
+type ChipProps = {
+    title: string;
+    onSelect: () => void;
+    description?: string;
+    icon: ReactNode;
+    isSelected?: boolean;
+};
+
+type ChipStyleProps = {
+    isSelected: boolean;
+};
+const chipStyles = prepareNativeStyle<ChipStyleProps>((utils, { isSelected }) => ({
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    alignSelf: 'flex-start',
+    height: 44,
+    backgroundColor: utils.colors.gray100,
+    borderWidth: utils.borders.widths.sm,
+    borderRadius: utils.borders.radii.round,
+    borderColor: utils.colors.gray400,
+    extend: {
+        condition: isSelected,
+        style: {
+            borderColor: utils.colors.forest,
+            borderWidth: utils.borders.widths.md,
+        },
+    },
+}));
+
+const chipTitleStyle = prepareNativeStyle<ChipStyleProps>((utils, { isSelected }) => ({
+    ...utils.typography.hint,
+    color: utils.colors.gray800,
+    extend: {
+        condition: isSelected,
+        style: {
+            ...utils.typography.callout,
+            color: utils.colors.forest,
+        },
+    },
+}));
+
+const chipDescriptionStyle = prepareNativeStyle(utils => ({
+    ...utils.typography.label,
+    lineHeight: 12,
```

Comment:
Because the components are overlapping in figma. I needed to get these texts closer together than they are with the default typography line height. They are label and callout in figma but I didn't find a way to overlap the boxes as it is in design.

Another option was to use negative margin (not supported by Android), but that would be too complicated imho, so I just opted out for a different line height. 
https://tppr.me/eKWrq

Reply:
Thanks for explanation, please add comment to code about this

Reply:
Added